### ISSUE TITLE
Microsoft syntax image preview broken

### ISSUE THREAD

**Issue by amwnag** [valentine195/obsidian-admonition#215]
While in live preview edit mode, images do not display properly when using Microsoft syntax instead of code blocks.

~~~
```ad-note
title: Note 1
![[Pasted image 20220209173348.png]]
```

> [!note:Note 2]
> ![[Pasted image 20220209173348.png]]
~~~

Here's the result:
![image](https://user-images.githubusercontent.com/86573349/157807159-d1c746e3-338f-446a-a36c-299ecf8760e5.png)

I saw an existing issue which might be closely related: #198

**Comment by valentine195** [comment]
That is correct, it is related. Embeds in LP are difficult, to say the least.

I am tagging this as won't-fix and closing because Obsidian has introduced callouts using the MSDoc syntax to insiders and I am removing the MSDoc syntax from the plugin going forward.
